packit_service/log_versions.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+# from packit import __version__ as packit_version
+
+from ogr import __version__ as ogr_version
+from packit_service import __version__ as ps_version
+from sqlalchemy import __version__ as sqlal_version
+from flask_restx import __version__ as restx_version
+from flask import __version__ as flask_version  # type: ignore
+
+
+# For some reason mypy has an issue with flask's __version__ even though it works completely fine.
```

Comment:
This comment is about a line which is 3 lines above. Usually comments tend to belong to lines that follow. Please move this above the line that belongs to.

"some reason" and "works completely fine" contradict each other, and in this sentence they are pretty vague: it's not clear what the `mypy` error is the `type: ignore` tries to get rid of. It would help future readers to clarify that.

Reply:
Sorry, I should've been clearer. I'll make these changes today and make the comments more detailed.